.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,71 @@
+name: Bug Report
+description: File a bug report
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before reporting a crash here, please make sure you use the latest version of PurpurExtras.
```

Comment:
latest version of extras AND Purpur itself, as that's what's supported

Reply:
Improved it :)